recipe/meta.yaml

```diff
@@ -12,16 +12,15 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1002
-  skip: true  # [not py2k]
```

Comment:
IIUC this was suppose to skip Python 2. IOW build on Python 3 only. Is that right @Jhsmit? If so, we can drop this since conda-forge no longer builds packages for Python 2.

Reply:
As far as I remember you authored this file (several years ago) and I think there never was python 3 support until now. Which seems strange. But at the moment PymageJ is python 3 only